tests/platform_tests/api/test_psu_fans.py

```diff
@@ -257,11 +263,17 @@ def test_get_fans_target_speed(self, duthosts, enum_rand_one_per_hwsku_hostname,
         self.assert_expectations()
 
     def test_get_fans_speed_tolerance(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platform_api_conn):
-
+        duthost = duthosts[enum_rand_one_per_hwsku_hostname]
         for j in range(self.num_psus):
             num_fans = psu.get_num_fans(platform_api_conn, j)
-
+            fans_skipped = 0
```

Comment:
here too, Is this new variable "fans_skipped" really needed ?

Reply:
good catch! removing this